### ISSUE TITLE
versions limit of 0 outputs error

### ISSUE THREAD

**Issue by Sporif** [herzrasen/pkghist#12]
I was trying to get a list of uninstalled packages without any details and came across this bug.  The output of `pkghist --limit 0` is:

```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:347:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

**Comment by herzrasen** [comment]
Thank you for reporting this. I pondered about it for a while and came to the conclusion that I'd rather restrict `limit` to `integers > 0` and implement an option named `--no-details`. 
You could then call `pkghist --removed-only --no-details` to get all packages that are currently uninstalled.

**Comment by herzrasen** [comment]
This is implemented in v0.2.0
